Functions/Assertions/PesterThrow.ps1

```diff
@@ -51,7 +51,7 @@ function PesterThrow([scriptblock] $ActualValue, $ExpectedMessage, $ErrorId, [sw
 
 function Get-DoMessagesMatch($ActualValue, $ExpectedMessage, $ExpectedErrorId) {
     if ($ExpectedMessage -eq "") { return $false }
-    return $ActualValue.Contains($ExpectedMessage)
+    return $ActualValue -like "*$ExpectedMessage*"
```

Comment:
Might be some potential for bugs here if `$ExpectedMessage` contains special characters.  Maybe this would be safer:

```posh
return $ActualValue -match [regex]::Escape($ExpectedMessage)
```